docs/architecture/decisions/0014-dotgov-content-organization.md

```diff
@@ -0,0 +1,19 @@
+# 14. Use existing federalist site for content, Django app for registrar functionality
+
+Date: 2022-10-04
+
+## Status
+
+Proposed
+
+## Context
+
+Currently web infrastructure for the dotgov program is split between a [home site hosted on Cloud.gov Pages](https://home.dotgov.gov/) and the registrar/registry [hosted by Verisign](https://domains.dotgov.gov/dotgov-web/). The two sites provide very different experiences, confusingly different domains, and don’t share the same technical infrastructure. When we move off Verisign we have the opportunity to rethink the relationship between these two sites.
```

Comment:
> When we move off Verisign we have the opportunity to rethink the relationship between these two sites.

"As we award a new contract that formally splits the registry/registrar functions, we have an opportunity to reimagine how users learn about .gov _and_ how they register and manage domains."